### PR TITLE
Update release dates for v23.27

### PR DESCRIPTION
**Pull request by gonzalemario**

### CODE REVIEW

File: RELNOTES.md [hunk 1]
```diff
@@ -16,11 +23,24 @@
...
 
+- TPA-637 Allow a different Barman user when connecting to servers
+
 ### Minor changes
 
-- TPA-634 Support Oracle Linux 7 and 8 on the Docker platform
+- TPA-467 Explicit permisions when creating filesystem objects
 
-  Support for AWS platform is underway.
+- TPA-609 Not relying on Makefile any more to install tpaexec from source
```
Comment (jdnevin) [inline RELNOTES.md:32]: Could be worded "No longer rely on Makefile to install tpaexec from source"

File: RELNOTES.md [hunk 2]
```diff
@@ -16,11 +23,24 @@
   version, if applicable. `tpaexec deploy` will then apply these changes
   to the servers in the cluster.
 
+- TPA-637 Allow a different Barman user when connecting to servers
+
 ### Minor changes
 
-- TPA-634 Support Oracle Linux 7 and 8 on the Docker platform
+- TPA-467 Explicit permisions when creating filesystem objects
```
Comment (jdnevin) [inline RELNOTES.md:30]: Should it add that this was for changing mode octals to strings, per Ansible `risky-octal` linting?

**Review by jdnevin — DISMISSED** [review]

File: RELNOTES.md [hunk 3]
```diff
@@ -16,11 +23,21 @@
...
 
-  Support for AWS platform is underway.
+- TPA-609 No longer rely on Makefile to install tpaexec from source
+
+- TPA-616 Allow deployment regardless of where dependencies originated
+
+- TPA-618 Generate a primary_slot_name on primary for efm
+
+  Generate a primary_slot_name also on primary node to be used in case of
+  switchover, to ensure the switched primary will have a physical slot on
```
Comment (martinmarques) [inline RELNOTES.md:39]:
```suggestion
  switchover, to ensure the demoted primary will have a physical slot on
```

File: RELNOTES.md [hunk 4]
```diff
@@ -29,16 +46,18 @@
   reconfigure command to prepare for major PGD upgrade. Improve default
   value when no previous config exist.
 
-- TPA-618 Generate a primary_slot_name on primary for efm
+- TPA-631 Warn if existing cluster are stil using 2q repos
```
Comment (martinmarques) [inline RELNOTES.md:49]:
```suggestion
- TPA-631 Warn if existing clusters are still using 2q repos
```

File: RELNOTES.md [hunk 5]
```diff
@@ -29,16 +46,18 @@
...
 
-  Generate a primary_slot_name also on primary node to be used in case of
-  switchover, to ensure the switched primary will have a physical slot on
-  the new primary.
+- TPA-634 and TPA 483 Support Oracle Linux 7, 8 and 9 on Docker platform
+
+  Support for AWS platform is underway.
```
Comment (martinmarques) [inline RELNOTES.md:53]: This seems like an unrelated line. What exactly does it mean?

File: RELNOTES.md [hunk 6]
```diff
@@ -29,16 +46,18 @@
...
 
 ## Bugfixes
 
 - TPA-560 Fix some patroni warnings
 
+- TPA-629 Avoid OOM condition by reducing maintenance_work_mem by default
```
Comment (martinmarques) [inline RELNOTES.md:59]:
```suggestion
- TPA-629 Avoid OOM condition on a running Postgres instance by reducing maintenance_work_mem by default
```

**Review by martinmarques — DISMISSED** [review]

**Review by haroon-github — APPROVED** [review]
